crates/subspace-networking/src/utils/piece_receiver.rs

```diff
@@ -35,19 +35,36 @@ pub trait PieceValidator: Sync + Send {
     ) -> Option<Piece>;
 }
 
+/// Stub implementation for piece validation.
+pub struct NoPieceValidator;
```

Comment:
It looks good for the associated types, especially for multiple default values but doesn't seem so for the regular generics.

```
impl pallet_timestamp::Config for Runtime {
	/// A timestamp: milliseconds since the unix epoch.
	type Moment = u64;
	type OnTimestampSet = ();
	type MinimumPeriod = ConstU64<{ SLOT_DURATION / 2 }>;
	type WeightInfo = ();
}
```

IMHO the next line is confusing: `CurrencyAdapter<Balances, ()>;`